src/components/SettingsSection.vue

```diff
@@ -39,6 +39,10 @@
       <RenameWallet
         v-if="setting === SETTINGS.RENAME"
         data-test="settings-rename"
+      />
```

Comment:
This is related with RenameWallet but affects DeleteWallet. There isn't margin between sections and RenameWallet has an extra left padding that should  be removed. Moreover, in full screen doesn't grow.

![image](https://user-images.githubusercontent.com/24905681/129556258-e339a289-2e1b-4971-b282-6110d8a2062c.png)

![image](https://user-images.githubusercontent.com/24905681/129556288-487542c4-1358-4010-bf8c-8b5fd7cf792d.png)
![image](https://user-images.githubusercontent.com/24905681/129556317-7cbb0153-c072-4afa-a67c-b955c179b4a2.png)